apps/perf-test/src/scenarios/DetailsRowFast.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { DetailsRow, IColumn, Selection, SelectionMode } from 'office-ui-fabric-react';
+
+// tslint:disable-next-line:typedef
+const Items = Array.from({ length: 10 }, (n, i) => ({
+  key: `Item ${i}`,
+  name: `Item ${i}`,
+  modified: new Date().toString(),
+  shared: 'Private',
+  size: `${Math.round(Math.random() * 1000) / 10}KB`
+}));
+
+const selection = new Selection();
+selection.setItems(Items);
+
+const Columns: IColumn[] = [
+  { key: 'a', name: 'Name', fieldName: 'name', minWidth: 200, maxWidth: 400 },
+  { key: 'b', name: 'Last modified', fieldName: 'modified', minWidth: 200, maxWidth: 400 },
+  { key: 'c', name: 'Shared', fieldName: 'shared', minWidth: 300, maxWidth: 300 },
+  { key: 'c', name: 'Size', fieldName: 'size', minWidth: 300, maxWidth: 300 }
+];
+
+const scenario = (
+  <DetailsRow itemIndex={0} item={Items[0]} columns={Columns} selection={selection} selectionMode={SelectionMode.single} useFastIcons />
+);
+
+export default scenario;
```

Comment:
You can optionally give this scenario a more readable name in [scenarioNames.js](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/apps/perf-test/src/scenarioNames.js). I'd love to find a way to define it in here, but since the build compiles an `index.html` the `perf-test` script can't easily extract it.